python/cudf/cudf/pylibcudf_tests/conftest.py

```diff
@@ -29,3 +31,30 @@
 )
 def pa_type(request):
     return request.param
+
+
+@pytest.fixture(
+    scope="session",
+    params=[
+        pa.int64(),
+        pa.float64(),
+        pa.uint64(),
+    ],
+)
+def numeric_pa_type(request):
+    return request.param
+
+
+@pytest.fixture(
+    scope="session", params=[opt.value for opt in plc.types.Interpolation]
```

Comment:
Why do you need to unpack the value? I think you want
```suggestion
    scope="session", params=[opt for opt in plc.types.Interpolation]
```
or am I misunderstanding something? The interp object should be passed straight through to pylibcudf calls.